projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -6523,7 +6522,7 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         let res = !this.nativeElement.parentElement ||
             this.nativeElement.parentElement.clientHeight === 0 ||
             this.nativeElement.parentElement.clientHeight === renderedHeight;
-        if (!isChromium()) {
+        if (!this.platform.isChromium && !this.platform.isFirefox) {
```

Comment:
@MayaKirova If possible, remove the method from the `selection.service`. I found only one other reference to this method inside the selection service and it can also be replaced with the platform util property check.